build.sbt

```diff
@@ -90,7 +90,7 @@ libraryDependencies ++= jacksonOverrides ++  Seq(
     "com.typesafe.play" %% "play-json" % "2.9.4",
     "com.typesafe.play" %% "play-iteratees" % "2.6.1",
     "org.julienrf" %% "play-json-derived-codecs" % "10.1.0",
-    "org.scalatest" %% "scalatest" % "3.2.15" % "test"
+    "org.scalatest" %% "scalatest" % "3.2.15" % Test
```

Comment:
Just curious, because I can see some sbt files use `Test` and others `"test"`, which I think [the docs](https://www.scalatest.org/user_guide/using_scalatest_with_sbt) also suggest that - does this make any practical difference?

Reply:
The value of `Test` is just the string `"test"` - it just means that if you mistype the name of a variable, the compiler will tell you straight away, but if you're typing the string, it might not be so obvious!